cloudbuild-e2e.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+# create a GKE cluster
+- name: 'gcr.io/cloud-builders/gcloud'
+  id: CreateCluster
+  args:
+  - 'container'
+  - 'clusters'
+  - 'create'
+  - '${TRIGGER_NAME}-${SHORT_SHA}'
+  - '--zone=${LOCATION}'
```

Comment:
how many nodes does the test require?  IIRC, the default is 3 nodes, so it might be good to only have 2 or even 1 node.

Reply:
updated to 1 node